test/unit/spec/appWidgets/input/select2ObjectinputSpec.js

```diff
@@ -251,8 +251,16 @@ define([
                     return TestUtil.wait(1000);
                 })
                 .then(() => {
-                    const $select = $(node).find('select');
+                    nodeStructures.push(this.node.innerHTML);
+                    expect(nodeStructures[0]).not.toEqual(nodeStructures[1]);
+                    const $select = $(this.node).find('select');
                     $select.val('stuff').trigger('change');
+                    return TestUtil.wait(1000);
+                })
+                .then(() => {
+                    nodeStructures.push(this.node.innerHTML);
+                    expect(nodeStructures[0]).not.toEqual(nodeStructures[2]);
+                    done();
```

Comment:
I didn't want to dig into implementation too far so this just checks that the DOM has changed.